lang/java/reef-runtime-yarn/src/main/java/org/apache/reef/runtime/yarn/driver/YarnContainerManager.java

```diff
@@ -110,58 +107,80 @@
     this.trackingURLProvider = trackingURLProvider;
     this.rackNameFormatter = rackNameFormatter;
 
-
     this.yarnClient.init(this.yarnConf);
 
     this.resourceManager = AMRMClientAsync.createAMRMClientAsync(yarnRMHeartbeatPeriod, this);
     this.nodeManager = new NMClientAsyncImpl(this);
+
     this.jobSubmissionDirectory = jobSubmissionDirectory;
     this.reefFileNames = reefFileNames;
     this.progressProvider = progressProvider;
+
     LOG.log(Level.FINEST, "Instantiated YarnContainerManager");
   }
 
-
+  /**
+   * RM Callback: RM reports some completed containers. Update status of each container in the list.
+   * @param completedContainers list of completed containers.
+   */
   @Override
-  public void onContainersCompleted(final List<ContainerStatus> containerStatuses) {
-    for (final ContainerStatus containerStatus : containerStatuses) {
-      onContainerStatus(containerStatus);
+  public void onContainersCompleted(final List<ContainerStatus> completedContainers) {
+    for (final ContainerStatus containerStatus : completedContainers) {
+      this.onContainerStatus(containerStatus);
     }
   }
 
+  /**
+   * RM Callback: RM reports that some containers have been allocated.
+   * @param allocatedContainers list of containers newly allocated by RM.
+   */
   @Override
-  @SuppressWarnings("checkstyle:hiddenfield")
-  public void onContainersAllocated(final List<Container> containers) {
+  public void onContainersAllocated(final List<Container> allocatedContainers) {
 
-    // ID is used for logging only
-    final String id = String.format("%s:%d",
-        Thread.currentThread().getName().replace(' ', '_'), System.currentTimeMillis());
+    String id = "NULL_ID"; // ID is used for logging only
```

Comment:
The idea was that this value should never show up. I'll change it back to `null`.